internal/util/csiconfig.go

```diff
@@ -48,6 +48,10 @@ type ClusterInfo struct {
 		NetNamespaceFilePath string `json:"netNamespaceFilePath"`
 		// SubvolumeGroup contains the name of the SubvolumeGroup for CSI volumes
 		SubvolumeGroup string `json:"subvolumeGroup"`
+		// KernelMountOptions contains the kernel mount options for CephFS volumes
```

Comment:
Extract the entire CephFS struct into a independent type.
That should help writing unit test cases instead of defining the type each time.

Reply:
Done.